http/client_acp.go

```diff
@@ -33,7 +35,8 @@ func (c *Client) AddPolicy(
 	methodURL := c.http.baseURL.JoinPath("acp", "policy")
 
 	addPolicyRequest := AddPolicyRequest{
-		Policy: policy,
+		Policy:    policy,
+		CreatorID: creator,
```

Comment:
We discussed this in the standup. The main concern was that identity is required for this function but optional for all others. Most of us thought keeping it a param was the better option, but happy to continue the discussion here.

Reply:
Thanks for the update, just wondering though that how does a user then specify this through JWT ?

Reply:
> We discussed this in the standup. The main concern was that identity is required for this function but optional for all others.

Shahzad has a point here, I don't think the conversation convered the HTTP client flow per-se. The parameter can be left in for the `client.Store` interface consistency, but in general for the HTTP client it should be using a Auth Header to communicate the identity of the person making the request, which is then authenticated/verified on the DB (eg in the middleware).

Obviously at the moment we don't do any authentication/verification of the identity so the middleware would just make sure that the provided `AddPolicyRequest` creator matches the provded Auth Header creator.

Reply:
> but in general for the HTTP client, it should be using a Auth Header to communicate the identity of the person making the request, which is then authenticated/verified on the DB (eg in the middleware).

We already do this for all other methods.

Because this method has an explicit identity / creator param it should be transmitted this way. If we want to use the context identity we should remove the param from the client interface.

Reply:
I thought about this a little. I think we can keep it the way it is now (without the auth header). Reworking it should be easy if need be

Reply:
@shahzadlone we decided in the standup to move it to the header and change the API method params.